src/main/java/com/github/JanLoebel/jsonschemavalidation/provider/DefaultJsonSchemaProvider.java

```diff
@@ -44,8 +42,12 @@ private InputStream createInputStream(String url) {
                 // TODO validate this
                 return new FileInputStream(new File(url));
             }
+            if (url.toLowerCase().startsWith("classpath:")) {
```

Comment:
ResourceUtils.getFile("classpath://") already is able to load classpath resources. Why do you want to use the "ClassPathResource" directly?

Reply:
I've tried various forms like below and none of them work, especially when you run out of IDE (compile and run via `java -jar`)
* `classpath://jsonschema/book.json`
* `classpath:jsonschema/book.json`
* `classpath:/jsonschema/book.json` 
* `classpath:///jsonschema/book.json` 

Then I've read that to correctly load from class path you need to use `ClassPathResource` and Spring will search it around.

Reply:
BTW, I use Gradle to build and run my application, so I've tested your module using it. In gradle I use `org.springframework.boot` plugin and the latest Spring

Reply:
```
java.io.FileNotFoundException: class path resource [//jsonschema/book.json] cannot be resolved to absolute file path because it does not exist
	at org.springframework.util.ResourceUtils.getFile(ResourceUtils.java:177) ~[spring-core-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at com.github.JanLoebel.jsonschemavalidation.provider.DefaultJsonSchemaProvider.createInputStream(DefaultJsonSchemaProvider.java:49) ~[main/:na]
	at com.github.JanLoebel.jsonschemavalidation.provider.DefaultJsonSchemaProvider.loadSchema(DefaultJsonSchemaProvider.java:15) ~[main/:na]
	at com.github.JanLoebel.jsonschemavalidation.provider.CacheableJsonSchemaProvider.loadSchema(CacheableJsonSchemaProvider.java:18) ~[main/:na]
	at com.github.JanLoebel.jsonschemavalidationexample.config.MyCustomJsonSchemaProvider.loadSchema(MyCustomJsonSchemaProvider.java:25) ~[main/:na]
	at com.github.JanLoebel.jsonschemavalidation.advice.JsonValidationRequestBodyControllerAdvice.beforeBodyRead(JsonValidationRequestBodyControllerAdvice.java:50) ~[main/:na]
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyAdviceChain.beforeBodyRead(RequestResponseBodyAdviceChain.java:92) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodArgumentResolver.readWithMessageConverters(AbstractMessageConverterMethodArgumentResolver.java:204) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.readWithMessageConverters(RequestResponseBodyMethodProcessor.java:158) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.resolveArgument(RequestResponseBodyMethodProcessor.java:131) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.method.support.HandlerMethodArgumentResolverComposite.resolveArgument(HandlerMethodArgumentResolverComposite.java:121) ~[spring-web-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.method.support.InvocableHandlerMethod.getMethodArgumentValues(InvocableHandlerMethod.java:167) ~[spring-web-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:134) ~[spring-web-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:106) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:888) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:793) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1040) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:943) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:660) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883) ~[spring-webmvc-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-embed-websocket-9.0.30.jar:9.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.3.RELEASE.jar:5.2.3.RELEASE]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.30.jar:9.0.30]
```

Reply:
ok, strange.. but fine, then keep the "ClassPathResource" :)